src/components/filterPatterns/popover/Popover.tsx

```diff
@@ -49,18 +51,18 @@ const Popover: FC<Props> = ({
           paddingY="2xl"
           shadow="xs"
           w="6xl"
+          ref={popoverContentRef}
         >
           <PopoverHeader paddingX="2xl">
-            {header ? (
-              header
-            ) : (
+            {header ?? (
```

Comment:
is there a reason you prefer this style? For me is harder to read, but might be personal preference. No blocker

Reply:
No, not really. I also prefer `||` operator but they do differ a bit in terms of falsy values. I mean in this case it probably does not matter. But this only returns right side of the operand if header is `null` or `undefined`. That's the only reason I prefer it, because it ignores falsy values.